modules/oe_paragraphs_gallery/config/install/field.field.paragraph.oe_gallery.field_oe_plain_text_long.yml

```diff
@@ -0,0 +1,18 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.paragraph.field_oe_plain_text_long
+    - paragraphs.paragraphs_type.oe_gallery
+id: paragraph.oe_gallery.field_oe_plain_text_long
+field_name: field_oe_plain_text_long
+entity_type: paragraph
+bundle: oe_gallery
+label: Description
+description: ''
+required: false
+translatable: true
```

Comment:
In other components field instances we don't make translatable, but in paragraphs we do, was just wondering about it

Reply:
Uhm it depends on what it is the field and the scope.

Reply:
Usually text fields are translatable, but "value" fields like boolean toggles or entity references are not.